<ui version="4.0" >
 <class>ConfigDialog</class>
 <widget class="QWidget" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>VCard File</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>File Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KUrlRequester" name="kcfg_Path" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis" >
         <string>Here you can select the vcard file which should be used as resource for the Akonadi collection.</string>
        </property>
        <property name="filter" >
         <string>*.vcf</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_ReadOnly" >
        <property name="whatsThis" >
         <string>With this option you can make sure that Akonadi only reads from the Collection.</string>
        </property>
        <property name="text" >
         <string>Read Only</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>kurlcomborequester</zorder>
     <zorder>kcfg_Path</zorder>
     <zorder>label_2</zorder>
     <zorder>kcfg_ReadOnly</zorder>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Auto save</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_Autosave" >
        <property name="whatsThis" >
         <string>This option will make sure that the changes are saved after the configured interval. A timer will be started when the first change is made and on time out all changes in between will be saved.
When you disable auto save then the changes will only be saved when Akonadi is stopped (e.g. on logout). You probably want to enable this option.</string>
        </property>
        <property name="text" >
         <string>Enable Auto Save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Save Interval (min.):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KIntNumInput" name="kcfg_AutosaveInterval" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>This spinbox configures the interval of the auto save timer in minutes. If you set it to 0 then each change will directly be written to the vcard file.</string>
        </property>
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>600</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
